packages/core/src/State.ts

```diff
@@ -289,6 +312,32 @@ export class State<
    * The next events that will cause a transition from the current state.
    */
   public get nextEvents(): Array<TEvent['type']> {
```

Comment:
Note that with the support for the event patterns (that has been added some time ago) - this might not always return what people would expect. The declared return type won't match the reality here when using those patterns.